.github/workflows/docker.yml

```diff
@@ -74,9 +76,10 @@ jobs:
           images: ghcr.io/txpipe/dolos
           tags: |
             type=raw,value=latest,enable={{is_default_branch}}
-            type=semver,pattern=v{{major}}
-            type=semver,pattern=v{{major}}.{{minor}}
-            type=semver,pattern=v{{version}}
+            type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') }}
+            type=semver,pattern=v{{major}},enable=${{ startsWith(github.ref, 'refs/tags/v') }}
+            type=semver,pattern=v{{major}}.{{minor}},enable=${{ startsWith(github.ref, 'refs/tags/v') }}
+            type=semver,pattern=v{{version}},enable=${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Are you sure we need for an explicit `enable` for semver tags?
I was under the impression that if the git tag is not present, the action will just skip semver.

Reply:
Unsure, but better safe than sorry